styles.css

```diff
@@ -1,24 +1,52 @@
 :root {
+    --hamburger-clr: #D84282;
     --menu-bar-width: 60px;
     --menu-bar-height: 10px;
 }
 
+body {
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #333;
+}
+
+.container {
+    display: inline-block;
+}
+
+.container:hover #menu {
+    background: transparent;
+    transition: background 200ms ease-in;
+}
+
+.container:hover #menu::before {
+    transform: translateY(20px) rotate(45deg);
+}
```

Comment:
Kan je de translateY niet weglaten bij zowel de ::before en ::after?
Dus bijvoorbeeld zo:
.container:hover #menu::after {
transition: .........
transform: rotate(-45deg)
}

Reply:
Ik heb met translate gewerkt in plaats van top/bottom. Als ik ze weghaal, komen de balkjes niet over elkaar tijdens de animatie.